src/Services/BookingService/BookingService.cs

```diff
@@ -124,9 +134,9 @@ public async Task<BookingProcessEntity> Get(string baseUrl, Page currentPage, st
                 bookingInformation.AppointmentEndTime = DateTime.Today.Add(nextAvailability.DayResponse.AppointmentTimes.First().EndTime);
             }
 
-            _pageHelper.SaveFormData(bookingInformationCacheKey, bookingInformation, guid, baseUrl);
-            var bookingInfo = new List<object> { bookingInformation };
-            return new BookingProcessEntity { BookingInfo = bookingInfo };
+            _pageHelper.SaveFormData(bookingInformationCacheKey, bookingInformation, sessionGuid, baseUrl);
```

Comment:
u might have already but van u verify this is okay when cachine is enabled, trying to remember why it was in a List<objects> previously, this should break somewhere no?

Reply:
ahh can ignore this, u have it inline now on the return